include/swift/AST/DiagnosticsSema.def

```diff
@@ -3148,6 +3148,8 @@ ERROR(assignment_bang_has_immutable_subcomponent,none,
 NOTE(change_to_mutating,none,
      "mark %select{method|accessor}0 'mutating' to make 'self' mutable",
      (bool))
+NOTE(masked_instance_variable,none,
+     "perhaps the instance variable of the same name was intended?", ())
```

Comment:
I didn’t really check but AFAIK we don’t have non-statement diagnostic notes, so I think it would be great if it could be rephrased into something like `add explicit self. to make $0 an instance variable which can be mutated`